src/components/document/document-content.tsx

```diff
@@ -18,7 +18,7 @@ import "./document-content.sass";
 
 interface IProps extends IBaseProps {
   context: string;
-  documentId?: string;
+  documentId?: string | undefined;
```

Comment:
```suggestion
  documentId?: string;
```
It's already declared as optional with the `?`. Adding `| undefined` should be redundant.